archaius2-core/src/main/java/com/netflix/archaius/DefaultAppConfig.java

```diff
@@ -205,46 +182,40 @@ public static Builder builder() {
         return new Builder();
     }
     
-    public static DefaultAppConfig createDefault() {
+    public static DefaultAppConfig createDefault() throws ConfigException {
         return new Builder().build();
     }
     
+    private final CascadingCompositeConfig     root;
     private final SimpleDynamicConfig          runtime;
-    private final CascadingCompositeConfig     override;
+    private final CascadingCompositeConfig     dynamic;
     private final CascadingCompositeConfig     library;
     private final DefaultConfigLoader          loader;
     private final PropertyFactoryDynamicConfigListener dynamicObserver;
     
-    public DefaultAppConfig(Builder builder) {
-        super(NAME);
+    public DefaultAppConfig(Builder builder) throws ConfigException {
+        super("APP_CONFIG");
```

Comment:
There is still a `NAME` constant with this value, why is the string used directly?